src/main/docs/guide/version-matrix.adoc

```diff
@@ -1,9 +1,9 @@
 - Micronaut Neo4j 6.x uses the Neo4j Java Driver 5.x that adds support for Java 17 runtime versions. To use it in a Micronaut Framework project the version is not required since that is exposed by the BOM.
 
-dependency::micronaut-neo4j-bolt[groupId="io.micronaut.neo4j"]
+dependency:micronaut-neo4j-bolt[groupId="io.micronaut.neo4j"]
 
 - Micronaut Neo4j 5.x uses Neo4j 3.x. It supports Java 8 and 11 runtime versions. To use it in a Micronaut Framework project the dependency needs to specify the `micronaut-neo4j-bolt` version.
 
-dependency::micronaut-neo4j-bolt[groupId="io.micronaut.neo4j", version="5.2.0"]
+dependency:micronaut-neo4j-bolt[groupId="io.micronaut.neo4j", version="5.2.0"]
```

Comment:
Should not we remove the java 8 and 11 section of the the docs targeting Micronaut Framework 4?

Reply:
Yes, I agree. I'll do that.